app/services/HCAService.js

```diff
@@ -101,8 +101,8 @@ class HCAService {
    */
   async pushAlertNewSubscribedAuthorities(count) {
     await PushNotification.localNotification({
-      title: languages.t('label.authorities_new_subcription_title', { count }),
-      message: languages.t('label.authorities_new_subcription_msg', { count }),
+      title: languages.t('label.authorities_new_subscription_title', { count }),
```

Comment:
make code changes in a separate PR please.

Reply:
The way it will work is. change only the en.js and the code in a separate PR. this will sync up to lokalise and will rename and should copy all the translations

Reply:
ok. will do